<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CBIFGeneration</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CBIFGeneration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="keepTempDir">
     <property name="text">
      <string>Keep temporary directory?</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Thumbnail Size:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size of Thumbnails: If width or Height is selected, the other is scaled to the original format.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="byImageWidth">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="imageWidth">
        <property name="suffix">
         <string> pixels</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>256</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="byImageHeight">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="imageHeight">
        <property name="suffix">
         <string> pixels</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>256</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QRadioButton" name="imageOriginalSize">
        <property name="text">
         <string>Keep Original Size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Thumbnail Image Interval:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSpinBox" name="imageInterval">
      <property name="specialValueText">
       <string/>
      </property>
      <property name="suffix">
       <string> seconds</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>900</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="generateGIF">
     <property name="text">
      <string>Generate GIF?</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="generateBIF">
     <property name="text">
      <string>Generate BIF?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>imageOriginalSize</tabstop>
  <tabstop>byImageWidth</tabstop>
  <tabstop>imageWidth</tabstop>
  <tabstop>byImageHeight</tabstop>
  <tabstop>imageHeight</tabstop>
  <tabstop>imageInterval</tabstop>
  <tabstop>generateBIF</tabstop>
  <tabstop>generateGIF</tabstop>
  <tabstop>keepTempDir</tabstop>
 </tabstops>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
